packages/Table/src/TableBody/TableBody.tsx

```diff
@@ -59,35 +54,29 @@ const TableBody = ({
             <div
               onClick={() => onClickRow({ ...allRows[index], index })}
               className="table-row"
-              key={uniqueId(allRows[index])}
-              data-testid={`tablerow-${index + 1}`}
+              key={`tablerow-${index}`}
```

Comment:
right, it should have an unique identifier

Reply:
Cool we can test with uuid but do you think using uuid might cause any performance issues? I remember a few months ago we removed them from as many places as we could to help with performance. 

I wonder if with Tables that infinitely scroll would it have to render have table row again because it would have a new uuid where as before the unique Id was generated from the element so was more likely to stay the same and potentially not rerendered? The other issue is that it might make using this component on the frontend harder as it will get rendered with one uuid on the server and another in the client so the server/client html will mismatch.

But if we make a note to test this as part of the pre release then we can test and follow up as a separate story if needed

Reply:
I've had same second though about uuid and replace it with nano-id.

As far as I noticed previously the id was generated using a custom function inside Utils call uniqueID implemented by David, which is not as performant or battle tests as nano-id.

But, yes, happy to test it and add a follow up story if nano-id causes any issues.